docs/cvl/methods.md

```diff
@@ -178,6 +190,63 @@ shown in the web report can indicate whether a summary was applied.
 
 ```
 
+(catch-unresolved-calls-entry)=
+### Catch unresolved-calls entry
+The catch unresolved-calls entry is a special type of summary declaration that instructs the Prover to replace calls to unresolved external function calls with a specific kind of summary, dispatch list.
+By default, the Prover will use an AUTO summary for unresolved function calls, but that may produce spurious counter examples. 
+The catch unresolved-calls entry let's the user refine the summary used for unresolved function calls.
+
+```{note}
+Only one catch unresolved-calls entry is allowed per a specification file```
+
+Catch unresolved-calls entry can only be summarized with a dispatch list summary (and a dispatch list summary is only applicable for a catch unresolved-calls entry). 
+A dispatch list summary directs the Prover to construct a summary that encapsulates the functionality of the method as a selector for multiple functions specified by their signatures. 
+The dispatch list will contain a list of patterns and the default summary to use in case no function matched the selector.
+The possible patterns are:
+1. Exact function - a pattern specifying both the primary or a linked contract, and the function signature.
+   Example: `C.foo(uint)`
+2. Wildcard contract - a pattern specifying the function signature to match this signature on the primary contract and all linked contracts.
+   Example: `_.bar(address)`
+3. Wildcard function - a pattern specifying the primary or a linked contract, and matches all external functions in specified contract.
+For the default summary the user can choose one of: `HAVOC_ALL`, `HAVOC_ECF`, `NONDET`.
+
+To better understand this functionality an example entry is useful.
+The following entry will specify three functions to route calls to:
+1. `C.foo(uint)`
+2. `Other.bar(address)`
+3. `C.baz(bool)`
+```cvl
+methods {
+   function _._ external => DISPATCH [
+      C.foo(uint),
+      _.bar(address), // Will resolve to all linked function with the signature "bar(address)", specifically Other.bar(address)
+      C._ // Will resolve to all functions in C, specifically C.foo(uint) and C.baz(bool)
+   ] default NONDET;
+}
+```
+Entry flags (`envfree`, `returns`, and `optional`) are not allowed on an unresolved-calls entry. 
```

Comment:
Entry flags are called otherwise. I think modifiers?

Reply:
I checked and `envfree` and `optional` are referred to as an annotation, but `returns` is a clause.  Fixing it accordingly